text/0095-add-latest-cf.md

```diff
@@ -0,0 +1,80 @@
+# Introduce latest cf
+
+- RFC PR: https://github.com/tikv/rfcs/pull/95
+- Tracking Issue: https://github.com/tikv/repo/issues/0000
+
+## Summary
+
+Add a cf (column family in RocksDB) named "latest" to store the latest version of keys in MVCC.
+
+## Motivation
+
+As we all know, currently TiKV stores all its data in RocksDB. It creates a cf (column family) named "write" to store all the available versions. The key format looks like following:
+
+```
+| key | version |
+```
+
+. Version is a 64 bits number and encoded in desc order (larger goes first).
+
+When reading a key with version v0, TiKV is expected to return the largest version v1 that v1 <= v0. Because TiKV has no idea what version is available, so it has to create an iterator and use version v0 to encode a seek key. If any key is hit, then the key should be the requested key.
+
+The procedure is straightforward but expensive. Creating an iterator in RocksDB is not free. Even for point get, an iterator is still necessary. And seek operation is also expensive, almost as expendsive as creating iterator. To avoid seeking too many times, we introduce `near_seek` to TiKV in the early days, which tries `next` several times before fallback to `seek`.
+
+As explained, the reason why seek is necessary is because TiKV has no idea what versions are available. Otherwise it can just use `get` to query the specific key, which is a lot cheaper.
+
+## Detailed design
```

Comment:
How about lightning, cdc, br, tiflash's compatibility?

Reply:
They are not part of TiKV. I would like to talk about them in another doc.